### ISSUE TITLE
[k8s] Consider submitting Charts to upstream helm/charts

### ISSUE THREAD

**Issue by aegershman** [YugaByte/yugabyte-db#1637]
In order to keep the yugabyte/charts repo in sync with upstream helm/charts, consider submitting yugaware/yugabyte charts to either the `stable` or `incubator` projects in upstream helm.

Inspiration drawn from [Concourse charts](https://github.com/concourse/charts), where they keep a fork of helm/charts and do chart development off of their fork's `master` && submit it to upstream `helm/charts` when they're ready to make it official. From what I understand, in this way, they're able to keep stable maintained in the official helm/charts, but a development fork in their github org.

(Disclaimer: I have never maintained a chart on the official helm/charts, only contributed to it / maintained internal work-only charts, so I don't know whether it's a total process nightmare to work on helm/charts or not).

**Comment by ramkumarvs** [comment]
@aegershman , yes that is in our roadmap items to do, but we would probably not get to it until after our 2.0
